### ISSUE TITLE
unable to use

### ISSUE THREAD

**Issue by boudiccas** [mlf176f2/tabbar-ruler.el#16]
I've just downloaded and installed tabbar-ruler but it is impossible to use as it locks up my emacs so nothing can be done in emacs. This is from my "init.org"

     ** tabbar ruler
     #+begin_src emacs-lisp
    (setq tabbar-ruler-global-tabbar t) ; If you want tabbar
     ;;(setq tabbar-ruler-global-ruler t) ; if you want a global ruler
    ;;(setq tabbar-ruler-popup-menu t) ; If you want a popup menu.
    ;;(setq tabbar-ruler-popup-toolbar t) ; If you want a popup toolbar
    ;;(setq tabbar-ruler-popup-scrollbar t) ; If you want to only show the
    ;;   ; scroll bar when your mouse is moving.
    (require 'tabbar)                                      
    (require 'tabbar-ruler)
    (tabbar-ruler-group-buffer-groups)
    ;;(tabbar-ruler-group-by-projectile-project)
   #+end_src
   [2015-04-21 Tue 18:52]

I am using "tabbar-20141109.143".

Its also shown this bug-trace -

````
Debugger entered--Lisp error: (invalid-function (lambda "Remove caches for tabbar-ruler." (mapc (function (lambda (frame) (modify-frame-parameters frame (quote ((tabbar-cache)))))) (frame-list))))
  tabbar-ruler-remove-caches()
  run-hooks(desktop-after-read-hook)
  desktop-read()
  #[nil "\304\211	\235\203
````

**Comment by mlf176f2** [comment]
What version of emacs are you using?

**Comment by boudiccas** [comment]
Matthew Fidler <notifications@github.com> writes:

> What version of emacs are you using?

emacs 24.5.50.1

Thanks
Sharon.
>
> On Tue, Apr 21, 2015, 1:55 PM boudiccas <notifications@github.com> wrote:
>
>> I've just downloaded and installed tabbar-ruler but it is impossible to
>> use as it locks up my emacs so nothing can be done in emacs. This is from
>> my "init.org"
>>
>> ** tabbar ruler
>> #+begin_src emacs-lisp
>> (setq tabbar-ruler-global-tabbar t) ; If you want tabbar
>> ;;(setq tabbar-ruler-global-ruler t) ; if you want a global ruler
>> ;;(setq tabbar-ruler-popup-menu t) ; If you want a popup menu.
>> ;;(setq tabbar-ruler-popup-toolbar t) ; If you want a popup toolbar
>> ;;(setq tabbar-ruler-popup-scrollbar t) ; If you want to only show the
>> ;; ; scroll bar when your mouse is moving.
>> (require 'tabbar)
>> (require 'tabbar-ruler)
>> (tabbar-ruler-group-buffer-groups)
>> ;;(tabbar-ruler-group-by-projectile-project)
>>
>> #+end_src
>> [2015-04-21 Tue 18:52]
>>
>> I am using "tabbar-20141109.143".
>>
>> Its also shown this bug-trace -
>>
>> Debugger entered--Lisp error: (invalid-function (lambda "Remove caches for tabbar-ruler." (mapc (function (lambda (frame) (modify-frame-parameters frame (quote
> ((tabbar-cache)))))) (frame-list))))
>> tabbar-ruler-remove-caches()
>> run-hooks(desktop-after-read-hook)
>> desktop-read()
>> #[nil "\304\211� \235\203�
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/mlf176f2/tabbar-ruler.el/issues/16>.
>>
>
> —
> Reply to this email directly or view it on GitHub.*
>

-- 
A taste of linux = http://www.sharons.org.uk
TGmeds = http://www.tgmeds.org.uk
Debian testing, fluxbox 1.3.7, emacs 24.5.50.1

**Comment by mlf176f2** [comment]
I don't have emacs 24.5, but this fix may work...?

**Comment by boudiccas** [comment]
Matthew Fidler <notifications@github.com> writes:

> I don't have emacs 24.5, but this fix may work...?
>
Unfortunately not. I updated from ELPA, and uncommented my
"tabbar-rule" settings, and then tangled again. It loaded
tabbar-rule but failed to load my desktop of all previous open
buffers. And the display of emacs was frozen, the cursor was unable
to be moved either up or down, left or right. And all tabs were
unclickable and unresponsive.

To tangle I do "C-c i", which tangles my init.org, writes init.el,
and then loads it.

Would it help if you had a copy of my "init.org" and "init.el" that
tangles it on starting emacs? Also a copy of my own home-grown
theme? If so, could you let me know a separate email address apart
From github to send them to?

Thanks
Sharon. 
-- 
A taste of linux = http://www.sharons.org.uk
TGmeds = http://www.tgmeds.org.uk
Debian 8.0, fluxbox 1.3.7, emacs 24.5.50.1

**Comment by mlf176f2** [comment]
This may be fixed with Issue #10 fix.
